### PR TITLE
[SHELL32][SHELL32_APITEST] Improve CommandLineToArgvW

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

This PR will reduce the failures of `CommandLineToArgvW` testcase of `shell32_apitest`.
JIRA issue: [CORE-17787](https://jira.reactos.org/browse/CORE-17787)

## Proposed changes

- Use `isspace` and `isblank` in `shell32!CommandLineToArgvW`.
- Strengthen `shell32_apitest:CommandLineToArgvW` testcase.

## TODO

- [x] Small tests.
- [x] Big tests.

## Screenshots

WinXP:
winxp
Successful.
Win2k3:
win2k3
Successful.
Win10:
win10
Successful.
BEFORE:
before
Some failures.
AFTER:
after
Successful.

### CODE REVIEW

**Comment by katahiromz** [comment]
Test KVM BEFORE: f73ac17 https://reactos.org/testman/compare.php?ids=85941
Test KVM AFTER: ef03896
Test VBox BEFORE: f73ac17 https://reactos.org/testman/compare.php?ids=85942
Test VBox AFTER: ef03896

File: dll/win32/shell32/wine/shell32_main.c [hunk 1]
```diff
@@ -130,16 +130,19 @@ LPWSTR* WINAPI CommandLineToArgvW(LPCWSTR lpCmdline, int* numargs)
...
+            s++;
     }
     else
     {
         /* The executable path ends at the next space, no matter what */
-        while (*s && *s!=' ' && *s!='\t')
+        while (*s && *s!=' ' && *s!='\t' && *s!='\r' && *s!='\n')
+            s++;
+        /* skip to the first argument, if any */
+        while (*s==' ' || *s=='\t' || *s=='\r' || *s=='\n')
```
Comment (learn-more) [inline dll/win32/shell32/wine/shell32_main.c:143]: If this also accepts stuff like `'\v'` and `'\f'` you could probably use `isspace` here.

**Review by learn-more — APPROVED** [review]
one thing that _could_ be changed, not sure if that's correct or not.

lgtm either way.

File: dll/win32/shell32/wine/shell32_main.c (hunk 1 above)
Comment (katahiromz) [reply dll/win32/shell32/wine/shell32_main.c]: OK.

**Comment by katahiromz** [comment]
Test KVM BEFORE: f73ac17 https://reactos.org/testman/compare.php?ids=85941
Test KVM AFTER: 69701c0
https://reactos.org/testman/compare.php?ids=85941,???

Test VBox BEFORE: f73ac17 https://reactos.org/testman/compare.php?ids=85942
Test VBox AFTER: 69701c0
https://reactos.org/testman/compare.php?ids=85942,???

**Comment by HBelusca** [comment]
@katahiromz With your latest additions there, does the test still pass OK on Win2k3 as well?

**Comment by katahiromz** [comment]
@HBelusca Yes.

**Pull request merged**